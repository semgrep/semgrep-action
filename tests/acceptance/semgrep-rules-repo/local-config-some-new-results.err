=== Running: SEMGREP_RULES="python/django/security/audit/xss/template-translate-as-no-escape.yaml" SEMGREP_BASELINE_COMMIT="HEAD~1" semgrep ci
Running `semgrep ci` without API token but with configs ('python/django/security/audit/xss/template-translate-as-no-escape.yaml',)
Scan environment:
  versions    - semgrep x.y.z on python x.y.z
  environment - running in environment git, triggering event is unknown

Fetching configuration from semgrep.dev
Scanning 6 files.
  Current version has 2 findings.

Skipping baseline scan, because all current findings are in files that didn't exist in the baseline commit.

Some files were skipped.
  Scan was limited to files changed since baseline commit.

Ran 1 rule on 6 files: 2 findings.
Ran 1 blocking rules, 0 audit rules, and 0 internal rules used for rule recommendations.
Has findings for blocking rules so exiting with code 1
