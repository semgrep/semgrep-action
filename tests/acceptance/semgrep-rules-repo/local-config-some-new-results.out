
Blocking Findings:

  python/django/security/audit/xss/template-translate-as-no-escape.html
     python.django.security.audit.xss.template-translate-as-no-escape
        Translated strings will not be escaped when rendered in a template. This leads to a
        vulnerability where translators could include malicious script tags in their translations.
        Consider using `force_escape` to explicitly escape a transalted text.

          2┆ {% translate "Hello world" as the_title %}
          3┆ <div>
          4┆ <h1>{{ the_title }}</h1>
          ⋮┆----------------------------------------
          8┆ {% trans "Hello world" as title %}
          9┆ <p>{{ title }}</p>

Blocking Rules Fired:
   python.django.security.audit.xss.template-translate-as-no-escape
